docs/react-testing-library/example-intro.mdx

```diff
@@ -72,15 +72,15 @@ See the following sections for a detailed breakdown of the test
 
```

Comment:
Initially I wanted to create a warning, but I noticed that we don't explicitly mention that the example uses MSW so I used a note.

```suggestion

:::note
The example uses [MSW (Mock Service Worker)](https://mswjs.io/) to mock HTTP requests.
MSW requires Node.js 18 or later.
:::
```

Reply:
Nevermind, it does mention it below the example.
Because I can't add a comment there, the lines at https://github.dev/kettanaito/testing-library-docs/blob/dbd489c8fa229dd43473daf7fd2d205c66fc3288/docs/react-testing-library/example-intro.mdx#L120-L122 should be removed.

```suggestion

:::note
We recommend using the [Mock Service Worker (MSW)](https://github.com/mswjs/msw) library
to declaratively mock API communication in your tests instead of stubbing
`window.fetch`, or relying on third-party adapters.

MSW requires Node.js 18 or later.
:::
```